sass/components/_footer.scss

```diff
@@ -0,0 +1,20 @@
+.footer {
+  &__social-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 10px;
+    padding: 20px;
+  }
+
+  &__social {
+    img {
+      height: 32px;
+      width: auto;
```

Comment:
Is the width and height necessary here if it is required for all the `<img>` tags?

Reply:
Mm, I guess not since I could just mark all of them as 32 x 32 in the tag rather than their actual sizes. I will fix this in a bit.